plugins/custom-cols.yaml

```
@@ -0,0 +1,34 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: custom-cols
+spec:
+  version: "v0.0.2"
```

```suggestion
  version: "v0.0.3"
```
The version should be "v0.0.3". I'll merge with v0.0.2, but keep in mind to update this in the future, otherwise krew will not distribute the new version.